scripts/setup_submodules

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# slightly modified from: https://stackoverflow.com/a/53899440/15053670
+
+set -euxo pipefail
+
+REPO_PATH=${1:-.}
+cd $REPO_PATH
+
+git init
+
+git config -f .gitmodules --get-regexp '^submodule\..*\.path$' |
```

Comment:
@gledis69 Could you add some more comment it is hard to grasp.

Reply:
Comments added, git doesn't detect them since only line 15 was selected for this comment.